packages/corvid-cli/src/utils/electron.js

```diff
@@ -152,14 +153,29 @@ async function openWindow(app, windowOptions = {}) {
       setTimeout(resolve, isDevTools ? 1000 : 0);
     }).then(async () => {
       const corvidConfig = await readCorvidConfig(".");
-
+      win.webContents
```

Comment:
please extract to a function:
const {santaVersion, editorVersion} = await getSantaVersions(win)
logger.addSessionData({...})